realm/realm-annotations-processor/src/main/java/io/realm/processor/ClassMetaData.kt

```diff
@@ -483,7 +483,7 @@ class ClassMetaData(env: ProcessingEnvironment, typeMirrors: TypeMirrors, privat
             if (!field.modifiers.contains(Modifier.FINAL)) {
                 continue
             }
-            if (Utils.isMutableRealmInteger(field)) {
+            if (Utils.isMutableRealmInteger(field) || Utils.isRealmList(field)) {
```

Comment:
Nitpick: Model classes have far more `RealmList`s  than `MutableRealmInterger`s so checking for RealmList first would be slightly faster.

Reply:
Have added this change 👍